app/assets/stylesheets/lti.scss

```diff
@@ -0,0 +1,14 @@
+// Style-sheet for lti
```

Comment:
I think we don't need any additional CSS for this and you can see the `app/views/lti/launch_error.html.erb` in which the container is already used but no additional css were required so delete the file and keep it as simple `container`